client/src/components/manageProjects/editableMeeting.js

```diff
@@ -76,6 +77,13 @@ const EditableMeeting = ({
         >
           DELETE
         </button>
+        <button
+          type="button"
+          className="create-form-button"
+          onClick={handleEventUpdate(eventId, {checkInReady: !checkInReady})}
+        >
+          ToggleCheckIn({!checkInReady ? "Yes" : "No"})
+        </button>
```

Comment:
`handleEventUpdate()` takes an `evendId`, a `values` object, and two other props unused here.

The `values` object is passed an obj with the opposite of `checkInReady`, and sent to the db updater fuction.